src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
```

Comment:
Use 'const' for 'totalCost' instead of 'let' since 'totalCost' is not being reassigned, but only modified.